Multisig/UI/UI Library/Styleguide/Fonts.swift

```diff
@@ -9,13 +9,48 @@
 import SwiftUI
 
 extension Font {
-    static let gnoHeadline = system(size: 17, weight: .semibold)
-    static let gnoSubhead = system(size: 15, weight: .bold)
-    static let gnoTitle3 = system(size: 20)
-    static let gnoNormal = custom("Averta Regular", size: 26)
-    static let gnoBody = system(size: 17)
-    static let gnoCallout = system(size: 16)
-    static let gnoCaption1 = system(size: 13, weight: .bold)
-    static let gnoCaption2 = system(size: 10, weight: .bold)
-    static let gnoFootnote = system(size: 13)
+    // ordered by increasing font size, then weight
+    static let gnoCaption3 =    system(size: 10, weight: .medium)
+    static let gnoCaption2 =    system(size: 10, weight: .bold)
+
+    static let gnoFootnote2 =   system(size: 13, weight: .medium)
+    static let gnoCaption1 =    system(size: 13, weight: .bold)
+
+    static let gnoSubhead =     system(size: 15, weight: .bold)
+
+    static let gnoCallout =     system(size: 16)
+
+    static let gnoBody =        system(size: 17, weight: .medium)
+    static let gnoHeadline =    system(size: 17, weight: .semibold)
+    static let gnoHeadline2 =   system(size: 17, weight: .bold)
+
+    static let gnoTitle3 =      system(size: 20)
+
+    static let gnoNormal =      custom("Averta Regular", size: 26)
+}
+
+// common text stylings
+extension Text {
```

Comment:
I still think it's better to define it as modifier, because in that case we can compose them, doesn't it ?

Reply:
the problem with applying modifiers to the text is that it creates new type, which is not Text, and so I can't compose some Text-related modifiers with a custom modifier 😞